indico/modules/events/contributions/blueprint.py

```diff
@@ -46,10 +46,14 @@
                  management.RHContributionsExportExcel, methods=('POST',))
 _bp.add_url_rule('/manage/contributions/contributions.pdf', 'contributions_pdf_export',
                  management.RHContributionsExportPDF, methods=('POST',))
-_bp.add_url_rule('/manage/contributions/book.pdf', 'contributions_pdf_export_book',
-                 management.RHContributionsExportPDFBook, methods=('POST',))
-_bp.add_url_rule('/manage/contributions/book-sorted.pdf', 'contributions_pdf_export_book_sorted',
-                 management.RHContributionsExportPDFBookSorted, methods=('POST',))
+
+# Tex-based exports
+_bp.add_url_rule('/manage/contributions/<uuid>/book-tex.zip', 'contributions_tex_export_book',
```

Comment:
I don't like mixing this UUID and contribution IDs on the same URL path level. I would go for a structure like this:

```
/manage/contributions/export/tex/  # for the form
/manage/contributions/export/tex/<uuid>  # for the download
```